pkgs/applications/science/misc/root/default.nix

```diff
@@ -120,7 +120,7 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-Drpath=ON"
-    "-DCMAKE_CXX_STANDARD=17"
+    "-DCMAKE_CXX_STANDARD=11"
```

Comment:
I would just remove this setting and let the compiler/build system decide what to use.